### PR TITLE
dashboard card

### PR DESCRIPTION
**Pull request by chevrechou**

### CODE REVIEW

File: pages/home.js [hunk 1]
```diff
@@ -2,6 +2,17 @@ import { useState } from "react";
...
+import { isAuthenticated, redirectHome } from "../lib/serverSideAuth";
+import MemorialCard from "../components/memorialCard";
+
+export async function getServerSideProps(ctx) {
+  const isAuth = await isAuthenticated(ctx);
+  if (!isAuth) return redirectHome(ctx);
+  return {
+    props: {},
+  };
+}
```
Comment (amanzanero) [inline pages/home.js:14]: we removed this in a previous PR. Try pulling from develop into your branch then push that commit

Comment (amanzanero) [reply pages/home.js:14]: oh you did that's weird

File: pages/home.js [hunk 2]
```diff
@@ -2,6 +2,17 @@ import { useState } from "react";
 import Head from "next/head";
 import Button from "../components/button";
 import DashNavbar from "../components/dashNavbar";
+import { isAuthenticated, redirectHome } from "../lib/serverSideAuth";
```
Comment (amanzanero) [inline pages/home.js:5]:
```suggestion
import { isAuthenticated, redirectHome } from "../lib/serverSideAuth";
```

File: pages/home.js (hunk 1 above)
Comment (amanzanero) [inline pages/home.js:14]:
```suggestion
```

Comment (chevrechou) [reply pages/home.js]: ..

**Review by amanzanero — APPROVED** [review]
lgtm

**Pull request merged**